test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleExecutionSpecs.java

```diff
@@ -0,0 +1,260 @@
+                        cryptoCreate("sender"),
+                        cryptoCreate("receiver"),
+                        scheduleCreate(
+                                "basicXfer",
+                                cryptoTransfer(
+                                        tinyBarsFromTo("sender", "receiver", transferAmount)
+                                )
+                        )
+                                .designatingPayer("payingAccount")
+                                .via("createTx")
+                ).when(
+                        scheduleSign("basicXfer")
+                                .withSignatories("sender", "payingAccount")
+                                .via("signTx")
+                                .hasKnownStatus(SUCCESS)
+                ).then(
+                        withOpContext((spec, opLog) -> {
+                            var signTx = getTxnRecord("signTx");
+                            var createTx = getTxnRecord("createTx");
+                            var triggeredTx = createTx.scheduled();
+
+                            allRunFor(spec, signTx, triggeredTx);
+
+                            Assert.assertEquals("Scheduled transaction be successful!", SUCCESS, triggeredTx.getResponseRecord().getReceipt().getStatus());
+
+                            Assert.assertEquals("Wrong consensus timestamp!",
+                                    signTx.getResponseRecord().getConsensusTimestamp().getNanos() + 1,
+                                    triggeredTx.getResponseRecord().getConsensusTimestamp().getNanos());
+
+                            Assert.assertEquals("Wrong transaction valid start!",
+                                    createTx.getResponseRecord().getTransactionID().getTransactionValidStart(),
+                                    triggeredTx.getResponseRecord().getTransactionID().getTransactionValidStart());
+
+                            Assert.assertEquals("Wrong record account ID!",
+                                    createTx.getResponseRecord().getTransactionID().getAccountID(),
+                                    triggeredTx.getResponseRecord().getTransactionID().getAccountID());
+
+                            Assert.assertTrue("Transaction not scheduled!",triggeredTx.getResponseRecord().getTransactionID().getScheduled());
+
+                            Assert.assertEquals("Wrong triggered transaction nonce!",
+                                    ByteString.EMPTY,
+                                    triggeredTx.getResponseRecord().getTransactionID().getNonce());
+
+                            Assert.assertEquals("Wrong schedule ID!",
+                                    createTx.getResponseRecord().getScheduleRef(),
+                                    triggeredTx.getResponseRecord().getScheduleRef());
+
+                            Assert.assertTrue("Wrong transfer list!", transferListCheck(triggeredTx, asId("sender", spec), asId("receiver", spec), transferAmount));
```

Comment:
I think that we should verify that there is a balance transfer list with a negative amount from the payer of the tx (the specified `payingAccount`